llvm/docs/CMake.rst

```diff
@@ -475,6 +475,17 @@ enabled sub-projects. Nearly all of these variable names begin with
 **LLVM_ENABLE_BINDINGS**:BOOL
   If disabled, do not try to build the OCaml bindings.
 
+**LLVM_ENABLE_DEBUGLOC_COVERAGE_TRACKING**:STRING
+  Enhances Debugify's ability to detect line number errors by storing extra
+  information inside Instructions, removing false positives from Debugify's
+  results at the cost of performance. Allowed values are `DISABLED` (default),
+  `COVERAGE`, and `COVERAGE_AND_ORIGIN`. `COVERAGE` tracks whether and why a
```

Comment:
Should we give an example performance impact for the `COVERAGE` mode here? I think it would help people evaluate how they use the feature.

Reply:
Makes sense - compile time tracker reckons that [currently](http://llvm-compile-time-tracker.com/compare.php?from=9e65dcac660723a06039c7e9b30f305b9b8ca652&to=f3f169f94a0c5e9cf2711438a5d932a8d8520f60&stat=instructions:u) this sits at a slowdown of about 0.05-0.1%, depending on configuration, so I'll put something to that effect in here.